joblib/memory.py

```diff
@@ -769,8 +769,22 @@ def clear(self, warn=True):
         self._write_func_code(func_code, first_line)
 
     def call(self, *args, **kwargs):
-        """ Force the execution of the function with the given arguments and
-            persist the output values.
+        """
+        Force the execution of the function with the given arguments.
```

Comment:
This should continue to say that the cache will be updated to the new output.
Else it might be confusing wether or not it does.

Reply:
Good call, added back in as a second line